src/pybind/mgr/dashboard/frontend/src/app/shared/forms/cd-validators.ts

```diff
@@ -103,6 +103,17 @@ export class CdValidators {
     );
   }
 
+  /**
+   * Validator that performs SSL certificate validation for rgw service.
+   * @returns {ValidatorFn} A validator function that returns an error map containing `pattern`
+   *   if the validation check fails, otherwise `null`.
+   */
+  static rgwCert(): ValidatorFn {
```

Comment:
It seems this method is validating a ".pem"-like certificate, so why not naming it `pemCert`?